addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComWindMessage.java

```diff
@@ -60,14 +61,17 @@ public static SubType fromByte(int input) throws RFXComUnsupportedValueException
 
     private static final List<RFXComValueSelector> SUPPORTED_INPUT_VALUE_SELECTORS = Arrays.asList(
             RFXComValueSelector.SIGNAL_LEVEL, RFXComValueSelector.BATTERY_LEVEL, RFXComValueSelector.WIND_DIRECTION,
-            RFXComValueSelector.WIND_SPEED);
+            RFXComValueSelector.AVG_WIND_SPEED, RFXComValueSelector.WIND_SPEED);

```

Comment:
How could both of us miss this 😉

The temperature fields were not added here.

See https://github.com/openhab/openhab2-addons/issues/2495#issuecomment-318463340

If you ( @mjagdis ) would have time to fix this that would be great, I have no opportunity because I m not near anything except for a tablet for at least one more week.